scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -367,6 +368,15 @@ class ScioContext private[scio] (val options: PipelineOptions,
   }
 
   /**
+   * Get an SCollection for an Protobuf file.
+   * @group input
+   */
+  def protobufFile[T: ClassTag](path: String)(implicit ev: T <:< Message): SCollection[T]
+  = pipelineOp {
+    this.objectFile(path)
```

Comment:
Better do a `.map()` first to convert `T` into `Array[Byte]` using `ProtoCoder` instead of default Kryo.